### ISSUE TITLE
scrollTo: true ineffective without special scrollHandler

### ISSUE THREAD

**Issue by ashnur** [shipshapecode/react-shepherd#266]
I can try to create a failing test later / so far I am just 90% sure this is probably an issue with the wrapper and not shepherd itself. The element I am trying to scroll to is a child of a different parent in a different component than where the start tour button is rendered. Just a guess so far, but I am thinking that when the selector is ran either the target element is not yet rendered, or by the time I press the button and it should scroll, the reference is lost due to React replacing the element.

**Comment by rwwagner90** [comment]
@ashnur any updates here?

**Comment by ashnur** [comment]
I couldn't figure out what was happening, so I ended up just wrapping popper.js myself with some custom elements and a scroller. If I have to configure everything, I can write everything, it's not more work.

**Comment by rwwagner90** [comment]
@ashnur we would be glad to help debug, but we would need some way to reproduce.

**Comment by ashnur** [comment]
Do you need help, or do I need help? :) Because this issue is more than a month old and I solved my problem without react-shepherd the same day I opened it. So I definitely do not need help since the problem was already solved about 888 hours ago :) 

https://stackblitz.com/edit/react-ctgk8g

**Comment by rwwagner90** [comment]
@ashnur creating your own implementation, instead of using a tour library, is not solving the problem. We have no way to know what your setup is, so your initial issue wasn't enough for us to look into it.

**Comment by rwwagner90** [comment]
@chuckcarpenter do you mind taking a look at this?

**Comment by ashnur** [comment]
> @ashnur creating your own implementation, instead of using a tour library, is not solving the problem.

:) I don't claim that my approach would solve everything react-shepherd does, but it did happen to get the result I was aiming for. Should I now decline myself this solution because you disagree with me about it?

> We have no way to know what your setup is, so your initial issue wasn't enough for us to look into it.

I believe that we are on the same page in this matter. All what happened was I ran into a problem trying to solve a problem, opened the issue, realized I have one another way to go about solving my original problem and forgot to close the issue when I finished that approach, and I am sorry about this.
